tmva/sofie/inc/TMVA/SOFIE_common.hxx

```diff
@@ -50,8 +49,34 @@ std::size_t ConvertShapeToLength(std::vector<size_t> shape);
 struct InitializedTensor{
    ETensorType type;
    std::vector<std::size_t> shape;
-   std::shared_ptr<void> data;
-   //void* data;
+   std::shared_ptr<void> data;     //! Transient
```

Comment:
small comments: 
we should maybe use ROOT naming conventions:
- starting with `f` the data members
- capitalise member functions
  `castSharedToPersistent`  -> `CastSharedToPersistent`

Reply:
Made the changes in names of data members & member functions of struct InitializedTensor